src/main/java/edu/ie3/datamodel/models/input/InputEntity.java

```diff
@@ -1,17 +1,11 @@
 /*
- * © 2021. TU Dortmund University,
+ * © 2024. TU Dortmund University,
  * Institute of Energy Systems, Energy Efficiency and Energy Economics,
  * Research group Distribution grid planning and operation
 */
 package edu.ie3.datamodel.models.input;
 
-import edu.ie3.datamodel.models.UniqueEntity;
-import java.util.UUID;
+import edu.ie3.datamodel.models.Entity;
 
-/** Functionless class to describe that all subclasses are input classes */
-public abstract class InputEntity extends UniqueEntity {
-
-  protected InputEntity(UUID uuid) {
-    super(uuid);
-  }
-}
+/** Interface to describe all input entities. */
+public interface InputEntity extends Entity {}
```

Comment:
This is necessary for the `MappingEntry`, because it is an `InputEntety` that does not have an `UUID`. I changed the previous class `InputEntity` into `UniqueInputEntity` which implements this interface.

Reply:
Well yes, but `MappingEntry` and `UniqueInputEntity` can now just extend `Entity`, right?

Reply:
This interface is used for the `InputDataSink` and the `InputEntityProcessor`. If we remove this interface we have to rewrite some of these classes.

Reply:
Got it, thanks.